setup.py

```diff
@@ -18,7 +18,7 @@
 import os
 from setuptools import setup
 
-with open("README.md", "r") as fh:
+with open("README.md", "r", encoding="utf_8") as fh:
```

Comment:
`utf_8` should be `utf-8`, no?

Reply:
utf-8 is alias of the utf_8 codec.

Both are valid.

https://docs.python.org/3/library/codecs.html#standard-encodings